study.md

```diff
@@ -87,7 +90,10 @@ Create an object literal named `vega` with the following properties and values.
 | solarMass | 2.135 |
 
 ```javascript
-// your answer here
+const vega = {
+  distance: 25.04;
```

Comment:
This line and the next will cause errors because key-value pairs need to be delimited by commas rather than semicolons.